Configure.pl

```diff
@@ -260,11 +260,13 @@ sub uniq {
 if ($args{'has-libtommath'}) {
     $defaults{-thirdparty}->{tom} = undef;
     unshift @{$config{usrlibs}}, 'tommath';
-    if (index($config{cincludes}, '-I/usr/local/include') == -1) {
-        $config{cincludes} = join(' ', $config{cincludes}, '-I/usr/local/include');
-    }
-    if (index($config{lincludes}, '-L/usr/local/lib') == -1) {
-        $config{lincludes} = join(' ', $config{lincludes}, '-L/usr/local/lib');
+    unless ($config{pkgconfig_works} && setup_native_library('libtommath')) {
```

Comment:
Are you saying you want it changed to ```if (!$config{pkgconfig_works} || !setup_native_library('libtommath'))``` due to preference for ```if``` over ```unless```, because the code within the block is the fallback if ```pkg-config``` isn't available or if a ```.pc``` file for libtommath isn't found. If ```setup_native_library``` succeeds then ```cincludes``` and ```lincludes``` should be already be populated.

If you want it changed due to preference for ```if```, I'm happy to do so. I just want to be sure that I have what you are asking for right.

Reply:
Okay will do. Thanks for clarifying :)